classes/game.rb

```diff
@@ -0,0 +1,14 @@
+require_relative './item'
+class Game
+  attr_accessor :multiplayer, :last_played_at
+
+  def initialize(multiplayer, last_played_at)
+    super(date)
+    @multiplayer = multiplayer
+    @last_played_at = last_played_at
+  end
+
+  def can_be_archive?
+    return true if Item.can_be_archive && @last_played_at > 2
+  end
+end
```

Comment:
This class could benefit from being a subclass of the Item class so that it can inherit its methods and instance variables. 

For example:

```
class Game < Item
...
end
```
It would also be helpful to include the id and publish_date arguments in the constructor since they are required by the parent class. This can be achieved by calling super() with these two arguments on line 6. 

For example:

```
class Game < Item
  def initialize(id, publish_date, multiplayer, last_played_at)
    super(id, publish_date)
    ...
  end
end
```
The @last_played_at variable represents the last time a game was played. To check if it was played over two years ago, you could find the difference between the current time and @last_played_at. For example:

```
def can_be_archived?
  super && (DateTime.now - @last_played_at).to_i / 365 > 2
end
```